app/javascript/packs/stylesheets/question.scss

```
@@ -0,0 +1,231 @@
+
+.sidebar__upvotes-count{
+  margin-top: 2px;
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 500;
+  font-size: 1.3em;
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+  -ms-word-break: break-all;
+  word-break: break-all;
+  word-break: break-word;
+  -ms-hyphens: auto;
+  -moz-hyphens: auto;
+  -webkit-hyphens: auto;
+  hyphens: auto;
+}
+
+.content__main{
+  padding: 5px 0 10px 5px;
+  width: 93%;
+  height: fit-content;
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 300;
+  font-size: 1em;
+}
+
+.post-details__footer{
+  padding-top: 10px;
+  width: 100%;
+  height: 90px;
+  border-top: 1pt solid rgba(0, 0, 0, 0.2);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.answer__footer{
+  height: 40px;
+}
+
+.answer__footer > .footer__author-container {
+  height: 40px;
+}
+
+.answer__footer > .footer__author-container > .author-container__avatar{
+  height: 40px;
+  width: 40px;
+}
+
+.footer__asked-at{
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 300;
+  font-size: 1em;
+}
+
+.footer__author-container{
+  margin-top: 10px;
+  width: 250px;
+  height: 50px;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.author-container__avatar{
+  height: 50px;
+  width: 50px;
+  -webkit-box-shadow: 0px 0px 11px 3px rgba(0,0,0,0.27);
+  -moz-box-shadow: 0px 0px 11px 3px rgba(0,0,0,0.27);
+  box-shadow: 0px 0px 11px 3px rgba(0,0,0,0.27);
+  background-size: cover;
+}
+
+.author-container__name{
+  font-family: 'Open Sans', sans-serif;
+  font-size: 1em;
+  margin-left: 10px;
+}
+
+
+.answers-container{
+  margin-top: 50px;
+  width: 100%;
+  height: 1000px;
+  display: flex;
+  flex-direction: column;
+}
+
+.answer-wrapper{
+  margin-top: 15px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+}
+
+.answer-details{
+  box-sizing: border-box;
+  width: 100%;
+  margin-top: -12px;
+  min-height: 100px;
+  height: fit-content;
+  background-color: white;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.actions-container{
+  width: 80px;
+  height: 30px;
+  align-self: flex-end;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.delete, .edit{
+  all: unset;
+  width: 30px;
+  height: 30px;
+  border: 1pt solid black;
+}
+
+.answer-details__bottom, .question-details__bottom{
+  clip-path: polygon(0% 15%, 0 0, 15% 0%, 85% 0%, 100% 0, 100% 15%, 100% 85%, 100% 47%, 86% 100%, 15% 100%, 0 47%, 0% 85%);
+  background-color: white;
+  width: 100%;
+}
+
+.answer-details__bottom{
+  height: 25px;
+}
+
+.question-details__bottom{
+  height: 50px;
+}
+
+.header{
+  -webkit-box-shadow: 1px 2px 2px 1px rgba(0,0,0,0.5);
+  -moz-box-shadow: 1px 2px 2px 1px rgba(0,0,0,0.5);
+  box-shadow: 1px 2px 2px 1px rgba(0,0,0,0.5);
+}
+
+.tox-tinymce{
+  height: 350px !important;
+  margin-top: 10px;
+}
```

Please add a newline to the end of the file :construction_worker_man:
```suggestion
}

```